docs/articles/life-cycle/skipping-tests.rst

```diff
@@ -166,3 +166,35 @@ tests and are not available at this point.
                     tests/test_passkey.py::test_skip__lambda (ldap) SKIPPED (Server is not built with password policy support)
                     tests/test_passkey.py::test_skip__function (ldap) SKIPPED (Server is not built with password policy support)
                     tests/test_passkey.py::test_skip__function_and_reason (ldap) SKIPPED (Server is not built with password policy support)
+
+Preferred topology
+===================
+
+Sometimes, you want to maintain the flexibility to run tests against all topologies but may
+not need to run all topologies all the time. It is possible to run the test once, skipping any
+additional topologies using ``pytest.mark.preferred_topology``.
+
+
+.. grid:: 1
+
+    .. grid-item-card::  Examples of @pytest.mark.preferred_topology()
+
+        .. tab-set::
+
+            .. tab-item:: @pytest.mark.preferred_topology()
+
+                .. code-block:: python
+
+                        @pytest.mark.preferred_topology("ldap")
```

Comment:
Don't work with names, we need to work mainly with objects here. Strings can be accepted as well to support the ad hoc topologies, but we want to use `KnownHost.LDAP` here as the main use case.